server/static/js/index/home_page_processing.js

```diff
@@ -0,0 +1,122 @@
+    let deviceType = {
+        1: "Orange Pi Lite",
+        2: "Orange Pi Zero",
+        3: "Orange Pi One",
+        4: "Orange Pi PC",
+        5: "Orange Pi Plus",
+        6: "Orange Pi 2G-IoT"
+    };
+
+    onlineStatus = onlineStatus === true ? "Online" : "Offline";
+
+    $(TABLE_DEVICES).append($row);
+    $row.append($("<input data-device-id type=\"hidden\">").val(rowDataDevice.id));
+    $row.append($("<td data-device-type-id>" + rowDataDevice.device_id + "</td>"));
+    $row.append($("<td data-device-type>" + deviceType[deviceTypeNumber] + "</td>"));
+    $row.children('[data-device-type]').attr('data-device-type', rowDataDevice.device_type);
+    $row.append($("<td data-device-last-address>" + rowDataDevice.last_address + "</td>"));
+    $row.append($("<td data-device-last-online>" + rowDataDevice.last_connected_at + "</td>"));
+    if (onlineStatus === "Online") {
+        $row.append($("<td data-device-status class='online'>" + onlineStatus + "</td>"));
+    } else {
+        $row.append($("<td data-device-status class='offline'>" + onlineStatus + "</td>"));
+    }
+}
+
+// Callback function, called on success, for reading users data
+function drawTableUsers (data) {
+    for (let index = 0; index < data.users.length ; index++) {
+        drawRowUsers(data.users[index]);
+    }
+}
+
+// The function of drawing user table rows
+function drawRowUsers(rowDataUsers) {
+    let $row = $('<tr class="table-row table-remove-row-user">');
+    let userEnabled = rowDataUsers.active;
+
+    userEnabled = userEnabled === true ? "Yes" : "No";
+
+    $(TABLE_USERS).append($row);
+    $row.append($("<input data-user-id type='hidden'>").val(rowDataUsers.id));
+    $row.append($("<td data-user-name data-first-name data-last-name>" +
+                "" + rowDataUsers.first_name + " " + rowDataUsers.last_name + "</td>"));
+    $row.children('[data-user-name]').attr('data-first-name', rowDataUsers.first_name);
+    $row.children('[data-user-name]').attr('data-last-name', rowDataUsers.last_name);
+    $row.append($("<td data-user-email>" + rowDataUsers.email + "</td>"));
+    $row.append($("<td data-user-last-login>" + rowDataUsers.last_login_at + "</td>"));
+    
+    if (userEnabled === "Yes") {
+        $row.append($("<td data-user-enabled class='online'>" + userEnabled + "</td>"));
+    } else {
+        $row.append($("<td data-user-enabled class='offline'>" + userEnabled + "</td>"));
+    }
+
+    $row.children('[data-user-enabled]').attr('data-user-enabled', rowDataUsers.active);
+
+    $row.append($("<td><div class='user-roles-container'></div></td>"));
+
+    for (let i = 0; i < rowDataUsers.roles.length; i++) {
+        $row.children('td:last-child').children('.user-roles-container').append($("<input data-user-role type='hidden'>").val(rowDataUsers.roles[i].name));
+    }
+}
+
+// Cleaning the device table
+function removeTableDevices () {
+    $(".table-remove-row-device").remove();
+}
+
+// Clearing the user table
```

Comment:
Function declarations should have proper JS docstring (if applicable), so consider changing broad comment to a docstring.